Makefile

```diff
@@ -170,6 +170,8 @@ check:
 	@! ineffassign . | grep -vF gossip/gossip.pb.go
 	@echo "errcheck"
 	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
+	@echo "returncheck"
+	@returncheck $(PKG)
```

Comment:
I don't currently use this, but I'd be in favor of keeping it (and where possible adding arguments to the other `make check` commands to possibly narrow their scope). If `make check` (or possibly a subset of it in a separate `make fastcheck` target) could be made fast enough when run on one file/package I could configure emacs/flycheck to run that instead of running tools like `errcheck` directly.